services/core-api/app/api/utils/access_decorators.py

```diff
@@ -15,6 +15,9 @@
 EDIT_SECURITIES = "core_edit_securities"
 MINESPACE_PROPONENT = "mds_minespace_proponents"
 EDIT_SUBMISSIONS = "core_edit_submissions"
+DATA_CLEANUP = "mds_data_cleanup"
```

Comment:
These are "user groups" and should not be modelled in our codebase. This should instead reflect the permission or capability necessary in the system. User group role assignment is achieved in Keycloak.